rhobs/alerting/data_plane/prometheus.gitops_sync_alerts.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-gitops-sync-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+    - name: gitops_sync_alerts
+      interval: 1m
+      rules:
+        - alert: GitOpsApplicationSyncErrors
+          expr: |
+            sum(argocd_app_info{health_status!="",sync_status=~"Synced"})
+            /
+            sum(argocd_app_info{health_status!="",sync_status!=""}) < 0.95
+          for: 5m
+          labels:
+            severity: warning
+          annotations:
+            summary: >-
+              Less than 95% of GitOps applications are not in synced state
```

I think the `not` should not be there.
```suggestion
              Less than 95% of GitOps applications are in synced state
```